src/components/Pagination/Pagination-test.js

```diff
@@ -4,11 +4,12 @@ import Icon from '../Icon';
 import Pagination from '../Pagination';
 import Select from '../Select';
 import SelectItem from '../SelectItem';
+import describeBreakingChangesXFeatures from '../../tools/describeBreakingChangesXFeatures';
 import { shallow, mount } from 'enzyme';
 
 jest.useFakeTimers();
 
-describe('Pagination', () => {
+describeBreakingChangesXFeatures('Pagination', () => {
```

Comment:
Should we put this in one of our jest setup files?

Reply:
Make sense - Done.